backend/database/versions/24_32f819760df4_add_curator.py

```diff
@@ -0,0 +1,30 @@
+"""add_curator
+
+Revision ID: 32f819760df4
+Revises: 21_c830a9e1f874
+Create Date: 2021-11-22 09:59:28.579807
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+from backend.corpora.common.corpora_orm import StrippedString
+
+revision = "24_32f819760df4"
+down_revision = "23_449eea8f5aeb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Nit: these comments can probably go

Reply:
> Looks good so far, but don't we need some more code to bind the new table?

By bind do you mean add curator_name to `Collection` in corpora_orm.py.